### PR TITLE
Add tink-cc@2.1.3 to BCR

### PR DESCRIPTION
**Pull request by morambro**
Adding BCR entry for https://github.com/tink-crypto/tink-cc

### CODE REVIEW

**Comment by bazel-io** [comment]
Hello @bazelbuild/bcr-maintainers, modules without existing maintainers (tink_cc) have been updated in this PR. Please review the changes.

File: modules/tink_cc/metadata.json [hunk 1]
```diff
@@ -0,0 +1,16 @@
+{
+    "homepage": "https://github.com/tink-crypto/tink-cc",
+    "maintainers": [
+        {
+            "email": "bcr-maintainers@bazel.build",
+            "name": "No Maintainer Specified"
```
Comment (meteorcloudy) [inline modules/tink_cc/metadata.json:6]: @morambro Do you mind listing yourself as a maintainer for this BCR module?

Comment (morambro) [reply modules/tink_cc/metadata.json]: Done

File: modules/tink_cc/2.1.2/presubmit.yml [hunk 2]
```diff
@@ -0,0 +1,10 @@
+build_targets: &build_targets
+- '@tink-cc//tink:tink_cc'
+
```
[the remark this answers is not in the document]
Comment (morambro) [reply modules/tink_cc/2.1.2/presubmit.yml]: Done.

File: modules/tink_cc/2.1.2/presubmit.yml [hunk 3]
```diff
@@ -1,10 +1,12 @@
...
-    build_targets: *build_targets
-  macos:
-    build_targets: *build_targets
-  ubuntu2004:
-    build_targets: *build_targets
+matrix:
+  platform:
+  - ubuntu2004
+  - macos
+  bazel: [6.x]
```
Comment (meteorcloudy) [inline modules/tink_cc/2.1.2/presubmit.yml:5]: Any reason to not support 7.x here?

Comment (meteorcloudy) [reply modules/tink_cc/2.1.2/presubmit.yml:5]: Does it work on Windows?

File: modules/tink_cc/2.1.2/MODULE.bazel [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Tink C++ Bazel Module definition."""
+module(
+    name = "tink_cc",
+    version = "2.1.0",
```
Comment (meteorcloudy) [inline modules/tink_cc/2.1.2/MODULE.bazel:18]: This version number doesn't match the actual version being checked in.

**Review by meteorcloudy — CHANGES_REQUESTED** [review]

File: modules/tink_cc/2.1.2/presubmit.yml (hunk 3 above)
Comment (morambro) [reply modules/tink_cc/2.1.2/presubmit.yml]: Added both. In our internal CI we do test Windows but not Bazel 7.x; I did a quick test and I didn't see issues with Bazel 7.x anyhow so enabling it.

**Review by meteorcloudy — APPROVED** [review]

**Pull request merged**